examples/inference/api_server_openai/query_http_requests.py

```diff
@@ -35,17 +35,21 @@
     help="Whether to enable streaming response",
 )
 parser.add_argument(
-    "--max_new_tokens", default=None, help="The maximum numbers of tokens to generate"
+    "--max_new_tokens", default=256, help="The maximum numbers of tokens to generate"
 )
 parser.add_argument(
-    "--temperature", default=None, help="The value used to modulate the next token probabilities"
+    "--temperature", default=0.2, help="The value used to modulate the next token probabilities"
 )
 parser.add_argument(
     "--top_p",
-    default=None,
+    default=0.7,
```

Comment:
Can we revoke the default value changes in this file? If these default values work for the mllm models, it makes sense to set them in `image_query_http_requests.py` as you have done. But this file more general so let's just use openai's default value?